notebooks/tutorials/cmle/sklearn_census.ipynb

```diff
@@ -0,0 +1,540 @@
+   "outputs": [],
+   "source": [
+    "! gsutil ls gs://$BUCKET_NAME/$OUTPUT_DIR"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step Four: Serving the Model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once the model is successfully created and trained, you can serve it. In CMLE, a model can have different versions. Therefore, in order to serve the model, you'll have to create two things.\n",
+    "\n",
+    "First, you will create a model with just a name and a region. This will somewht act as a container for diffrent versions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud ml-engine models create $MODEL_NAME --regions us-central1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After creating the model, you can create a version and point to the model that you created in the previous step:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!gcloud ml-engine versions create $MODEL_VERSION \\\n",
+    "  --model=$MODEL_NAME \\\n",
+    "  --framework=scikit-learn \\\n",
+    "  --origin=gs://$BUCKET_NAME/$OUTPUT_DIR \\\n",
+    "  --python-version=$PYTHON_VERSION \\\n",
+    "  --runtime-version=$RUNTIME_VERSION"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step Five: Prediction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the model and the version created in the last step, you are ready to use it to make some predictions:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "MODEL_NAME = %env MODEL_NAME\n",
+    "MODEL_VERSION = %env MODEL_VERSION\n",
+    "import googleapiclient.discovery\n",
+    "\n",
+    "instances = [\n",
+    " [50, 'Private', 160187, '8th', 5, 'Married-spouse-absent', 'Other-service',\n",
+    "  'Not-in-family', 'Black', 'Female', 0, 0, 16, 'Jamaica'],\n",
+    "    \n",
+    " [52, 'Self-emp-not-inc', 209642, 'HS-grad', 9, 'Married-civ-spouse',\n",
+    "  'Exec-managerial', 'Husband', 'White', 'Male', 0, 0, 48, 'United-States']\n",
+    "]\n",
+    "\n",
+    "service = googleapiclient.discovery.build('ml', 'v1')\n",
+    "\n",
+    "name = 'projects/{}/models/{}/versions/{}'.format(PROJECT_ID, MODEL_NAME, MODEL_VERSION)\n",
+    "\n",
+    "response = service.projects().predict(\n",
+    "        name=name,\n",
+    "        body={'instances': instances}\n",
+    "    ).execute()\n",
+    "\n",
+    "if 'error' in response:\n",
+    "    print(response['error'])\n",
+    "else:\n",
+    "    print(response['predictions'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step Six: Cleanup"
```

Comment:
After this section, add a section called "Next Steps" with links to more tutorials the user could follow, reference docs for the libraries/products used, etc.

Reply:
Added at the end. Although it now looks a bit strange. to have a section called "Next Steps" after the previous section which is "Step Six: Cleanup". It now feels the tutorial is incomplete and there are actual steps to take after the 6th step. Maybe we should name the last section as "Reference" or "What is Next" or something.

Reply:
With this tutorial having a couple optional sections, I think it might work best to just get rid of the numbering altogether.